portality/models/v2/journal.py

```diff
@@ -1121,4 +1121,25 @@ def query(self):
             "sort" : [
                 {"created_date" : {"order" : "desc"}}
             ]
-        }
\ No newline at end of file
+        }
+
+class JournalDiscontinuedDateQuery(object):
```

Comment:
Implemented but then appeared not to be needed. Is it worth leaving or should be scrapped? Not currently used.

Reply:
I suggest to scrap if not used